docs/BUILD.md

```diff
@@ -122,6 +132,15 @@ specific versions without polluting the user's normal workspace.
 
 The following are the output components from the build process and their purpose. There are two kinds of components: final, intended for actual direct usage, and interim, used to build the final components. Some interim components may be removed as part of the build finalization process.
 
+However, **every** build will create `rootfs.tar`. This contains the root filesystem, whose contents then are
+used to generate the final versions: live vs installer, qcow2 vs raw vs GCP, etc.
```

Comment:
In some cases rootfs.tar is the final e.g., I need a rootfs to upload and use to update devices.
So some rewording might be helpful.

Reply:
Updated, please take a look.